text/0000-nested-publish.md

```diff
@@ -119,9 +119,10 @@ Then you can `cargo publish` from within the parent directory `foo/`, and this w
 
 Two new possible values are added to the manifest.
 
-* The `package.publish` field allows `"nested"` as a value, in addition to existing `false` and `true`. This value affects `cargo publish` and nested dependencies as described below.
+* The `package.publish` or `workspace.package.publish` field allows `"nested"` as a value, in addition to existing `false` and `true`. This value affects `cargo publish` and nested dependencies as described below.
```

Comment:
I have mixed feelings on allowing `nested` to be inherited.  In part, I feel like its a weird default for the full workspace but I also appreciate leaving the individual building blocks and letting users decide what to do.